### ISSUE TITLE
Webpack Example does not build

### ISSUE THREAD

**Issue by hediet** [finos/perspective#1149]
### Steps to Reproduce:
 1. copy examples/webpack from e5b6e347333d9d3d22138a25e0992728e07eb8a7
 3. run `yarn` inside the copy so that it downloads perspective from npm
 4. `yarn start`

### Expected Result:
It serves the example.

### Actual Result:
```
ERROR in ./node_modules/regular-table/dist/less/material.less 11:0
Module parse failed: Unexpected character '@' (11:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
|
> @import (css) url("https://fonts.googleapis.com/css?family=Open+Sans");
|
| @row-height: 19px;
 @ ./node_modules/@finos/perspective-viewer-datagrid/dist/cjs/perspective-viewer-datagrid.js 1:137-185
 @ ./src/index.js
Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./node_modules/html-webpack-plugin/default_index.ejs] 376 bytes {0} [built]
    [./node_modules/lodash/lodash.js] 530 KiB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 472 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
i ｢wdm｣: Failed to compile.
```

### Environment:
Node v14, Windows 10

### Additional Context:
Seems like less needs to be configured. It would be cool if that wasn't necessary. I'm already using sass and don't want to install less too.

**Comment by texodus** [comment]
Thanks for the report - this is a recent regression of the `@finos/perspective-viewer-datagrid` plugin, which has a fix merged and will be released in v0.5.3

**Comment by texodus** [comment]
Fix has been released in 0.5.3
